webhook/main.go

```diff
@@ -67,6 +68,7 @@ func main() {
 	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
 		Namespace: namespace,
 		Scheme:    scheme,
+		HealthProbeBindAddress: ":8443",
```

Comment:
I think this will cause a conflict since the webhook server listens on this port so the health service can't bind:
```json
{
  "msg":"Manager exited non-zero","error":"listen tcp 0.0.0.0:8443: bind: address already in use",
  "stacktrace":"main.main\n\t/devworkspace-operator/webhook/main.go:109\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:203"
}
```

Reply:
Thanks @amisevsk! changed to 6789 :-)